src/components/BottomToolbarPrimaryAction/index.tsx

```diff
@@ -88,7 +88,7 @@ export class BottomToolbarPrimaryAction extends React.Component<BottomToolbarPri
       <View style={finalWrapperStyles}>
         {items.map((item, index) => {
           const iconMode = !!item.icon;
-          const finalStyles = styleReferenceBreaker(iconMode ? this.styles.itemIconStyle : this.styles.itemTextStyle, item.style);
+          const finalStyles = StyleSheet.flatten([iconMode ? this.styles.itemIconStyle : this.styles.itemTextStyle, item.style]);
```

Comment:
I started "working with" react-native this week, so I cannot claim any profound knowledge of their APIs, but this one seems like the perfect match for many of the sites ... essentially being a squasher of many style sources into one